libr/asm/asm.c

```diff
@@ -189,12 +189,22 @@ static void plugin_free(RAsmPlugin *p) {
 	}
 }
 
-R_API RAsm *r_asm_new() {
+R_API RAsm *r_asm_new(RNum *num) {
 	int i;
 	RAsm *a = R_NEW0 (RAsm);
 	if (!a) {
 		return NULL;
 	}
+	if (num) {
+		a->num = num;
+		r_num_ref (num);
+	} else {
+		a->num = r_num_new (NULL, NULL, NULL);
```

Comment:
there's no need for this, the rnum api works with NULL as instance, it will just not resolve any non-numeric name

Reply:
?? ... 908b0e042e935b2324aa699c6229adf83d47b5bd resolves this?